providers/whitelists-provider/actions.js

```diff
@@ -34,13 +34,15 @@ export const getWhitelist = createThunkAction(
       ),
     ])
       .then(
-        spread((annualResponse, gladResponse, viirsResponse, modisResponse) => {
+        // spread((annualResponse, gladResponse, viirsResponse, modisResponse) => {
+        spread((annualResponse) => {
           dispatch(
             setWhitelist({
+              // TODO: re-activate reponses
```

Comment:
Is this TO-DO something important to do right after TTC updates? Should we be aware of something else?

Reply:
@willian-viana nice catch! 

I vaguely remember a mention for these whitelists being disabled. 

After some debugging/testing, I infer that when development on these widgets was first started, we still didn't have the new TTC tables and the intersections were failing; causing the widgets to not render any data, hence the "temporarily" disabled. 

The table versions were updated about 2 weeks later by myself (and the ENV was switched to `preproduction`, where the new tables exist), but it looks like we forgot to re-enable the whitelists. 

I've reverted the commit that disables the whitelists after a bit of testing, it looks like everything works now with the whitelists enabled.